README.md

```diff
@@ -72,24 +72,38 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
+   - Enter the following parameters
+     - for **Scope name** use `access_as_user`
+     - Keep **Admins and users** for **Who can consent**
+     - in **Admin consent display name** type `Access TodoListService-NativeDotNet as a user`
+     - in **Admin consent description** type `Accesses the TodoListService-NativeDotNet as a user`
+     - in **User consent display name** type `Access TodoListService-NativeDotNet as a user`
+     - in **User consent description** type `Accesses the TodoListService-NativeDotNet as a user`
+     - Keep **State** as **Enabled**
+     - Select **Add scope**
 
 #### Register the client app (TodoListClient-NativeDotNet)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'TodoListClient-NativeDotNet' and select 'Native' as the *Application Type*.
-1. For the *Redirect URI*, enter `https://<your_tenant_name>/TodoListClient-NativeDotNet`, replacing `<your_tenant_name>` with the name of your Azure AD tenant.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
-   click on **Add**, then **Select an API**, and type `TodoListService-NativeDotNet` in the textbox. Then, click on  **Select Permissions** and select **Access 'TodoListService-NativeDotNet'**.
+1. In the  **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. Enter a friendly name for the application, for example `TodoListClient-NativeDotNet`.
+1. Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+1. For the *Redirect URI (optional)*, select "Public client (mobile & desktop)" from the dropdown, and enter the base URL for the sample. By default, this sample uses `https://localhost:44321/`.
+1. Click **Register** to create the application.
+1. In the succeeding page, Find the *Application (client) ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. Configure Permissions for your application. To do so, select **API Permissions** and select **Add a permission** under the *API Permissions* section.
+   - Ensure that the **My APIs** tab is selected
+   - In the list of APIs, select the API `TodoListService-NativeDotNet`.
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: `access_as_user`. Use the search box if necessary.

```

Comment:
[Information]
`user_impersonation` if you changed the name of the scope above (which I would not think is needed)